### ISSUE TITLE
ScreenShareAudio source for TrackToggle/MediaTrack doesn't work

### ISSUE THREAD

**Issue by multimokia** [livekit/components-js#366]
### Select which package(s) are affected

\@livekit/components-react

### Describe the bug

When creating a component to screenshare with audio, I expect to be able to use the `ScreenShareAudio` source as documented, likewise as it exists in the `Track.Source` enum. 

In practice however, this yields no `ParticipantTile` for the media share, nor does the track toggle have an icon.
It does allow me to select a media source when the component is rendered, however it does not seem to allow the button to function after that as clicking on it does nothing. It also does not seem to share it as it appears on neither the clientside nor the receiving end.

Is this feature unsupported, or am I doing something incorrectly?

### Reproduction

Component for sample:

```tsx
export default function Stream({ user_id, username, channel_id }: { user_id: string; username: string, channel_id: number }) {
  const token = useToken(
    process.env.NEXT_PUBLIC_LK_TOKEN_ENDPOINT,
    channel_id.toString(), {
      userInfo: {
        identity: user_id,
        name: username
      },
    }
  );

  return (
    <div className={'bg-gray-800'}>
      <div className='flex flex-row justify-center'>
        <LiveKitRoom
          video={true}
          audio={true}
          screen={{ audio: true }}
          token={token}
          serverUrl={process.env.NEXT_PUBLIC_LK_SERVER_URL}
          connect={true}
          className='flex flex-col w-full justify-center'
        >
          <RoomAudioRenderer/>
          <div className='ml-13 flex flex-row'>
            <ParticipantLoop>
              <ParticipantTile className={'w-11 mb-5 mr-4'}>
                <MediaTrack source={Track.Source.Camera} className={'rounded-xl mb-4'}/>
                <MediaTrack source={Track.Source.ScreenShareAudio} className={'rounded-xl'}/>
                <MediaTrack source={Track.Source.Microphone}/>
              </ParticipantTile>
            </ParticipantLoop>
          </div>
          <div className='flex flex-row justify-evenly'>
            <TrackToggle showIcon={false} className={'w-7 h-7 bg-grey-900 rounded-lg text-lg'} source={Track.Source.Microphone} initialState={false}>
              Mic
            </TrackToggle>
            <TrackToggle className={'w-7 h-7'} source={Track.Source.Camera} initialState={false}/>
            <TrackToggle className={'w-7 h-7'} source={Track.Source.ScreenShareAudio} initialState={false}>
              Screenshare + Audio
            </TrackToggle>
          </div>
        </LiveKitRoom>
      </div>
    </div>
  );
};
```

We have been working with Supabase in our app to get user ids and other info, but for the sake of simplicity and reproducability, I have somewhat refactored the above to be easily dropped into a React app with arbitrary values passed in.

The token endpoint follows the same design as the documentation, just adapted for a NextJS serverless function.
The livekit server running is the dev server on windows.

Ideally when loading a project with this component even as a single user, the site should initially prompt for a screen/application to share, however nothing will be even if the browser states it is sharing.

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (32) x64 AMD Ryzen 9 5950X 16-Core Processor
    Memory: 30.82 GB / 63.91 GB
  Binaries:
    Node: 19.7.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.1.2 - C:\Program Files\nodejs\npm.CMD

Firefox 111.0, Edge 111.0.1661.41
```

### Severity

serious, but I can work around it

### Additional Information

Note, workaround is simply foregoing screenshare with audio. While it's not ideal, if we must we will.

**Comment by lukasIO** [comment]
hi @multimokia 

the source `ScreenShareAudio` really only represents the audio track of a screen share and not the screen share itself. 
Because it's impossible to get the screen audio separately from the screen video, there's no support for it on the toggle. 

In order to get a screen share with audio, you would add capture options (landed [here](https://github.com/livekit/components-js/pull/356)) to the toggle. 
This would look like 
```tsx
<TrackToggle className={'w-7 h-7'} source={Track.Source.ScreenShare} captureOptions={{audio: true}} initialState={false}>
   Screenshare + Audio
</TrackToggle>
```

**Comment by lukasIO** [comment]
will close this as solved, as `v0.6.0` has a narrowed type and capture options that should make this clearer.

**Comment by multimokia** [comment]
Yep this did the trick thank you so much for your quick response.
